### PR TITLE
Fixing erroneous padding in the arm64 pipeline yaml file

### PR DESCRIPTION
**Pull request by nihemstr**

### CODE REVIEW

**Review by shiyi-peng — APPROVED** [review]
:shipit:

File: azurepipelines/adu-ubuntu-arm64-build.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 variables:
-  version.major: 0
-  version.minor: 7
-  version.patch: 0
-  version.pre-release: "rc1-public-preview"
+    version.major: 0
+    version.minor: 7
```
Comment (JeffMill) [inline azurepipelines/adu-ubuntu-arm64-build.yml:3]: >version [](start = 4, length = 7)

yaml is truly horrible. do you know if ADO pipelines support any other format?

**Review by JeffMill — APPROVED** [review]
:shipit:

File: azurepipelines/adu-ubuntu-arm64-build.yml (hunk 1 above)
Comment (nihemstr) [reply azurepipelines/adu-ubuntu-arm64-build.yml:3]: The official standard is yaml. It's wht all their examples use. If there is another one they do not make it obvious to find.

---
In reply to: [612795016](https://github.com/Azure/iot-hub-device-update/pull/40#discussion_r612795016) [](ancestors = 612795016)

**Pull request merged**